### DISCUSSION TITLE
Can't able to login to GITHub with 2 way authentication - Please delete MFA for my GIT account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#43824] [Mobile] [answered]
Hi All,

I have reset my mobile and laptop, So I lost my both 2-way authentication codes. I can’t be able to do login/rest password my GIT account,
Please help me to disable this 2-way authentication for GIT account, 
And I can be able to login with password, but 2-way authentication is not working. And also, I am not having other recovery method like SSL, Personal Token details

Thanks

**Comment by stevepopovich** [comment]
Hi @rajkumar-perumal-ext can you try following these steps to recover your account? https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

**Reply by ghost** [reply]
Hi @stevepopovich 
 Yes, I tried all those steps, It not works, I am not have Auth code Mobile app, Recovery code, SSL Key and Personal Token. So that, I can't able to processed with recovery process of my git account. Main password is working and getting email for reset password but SSL Key and Personal Token not having, So Can't able to process. 

Need to disable 2 Way Auth MFA, then I will login with my password

**Comment by ghost** [comment, marked as answer]
Resolved this issue by deleted my account email and name by using Git support link
https://support.github.com/contact/account

**Reply by stevepopovich** [reply]
Great, sorry I couldn't of been more help!

**Comment by mjanulaitis** [comment]
How can I reset my password when MFA is not working without deleting my account?
